crypto/argon2/blake2b.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Argon2 reference source code package - reference C implementations
+ *
+ * Copyright 2015
+ * Daniel Dinu, Dmitry Khovratovich, Jean-Philippe Aumasson, and Samuel Neves
+ *
+ * You may use this work under the terms of the Apache Public License 2.0.
+ * The terms of this licenses can be found at:
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * You should have received a copy of the license along with this
+ * software. If not, it may be obtained at the above URL.
+ */
+
+#ifndef __BLAKE2B_H
+# define __BLAKE2B_H
+
+# include <stdio.h>
+# include <stdlib.h>
+# include <string.h>
+
+# include "core.h"
+
+# include <openssl/evp.h>
+# include <openssl/objects.h>
+# include <openssl/crypto.h>
+# include <prov/blake2.h>
+
+static ossl_inline uint32_t load32(const uint8_t *src)
+{
+    const union {
+        long one;
+        char little;
+    } is_endian = { 1 };
+
+    if (is_endian.little) {
+        uint32_t w;
+        memcpy(&w, src, sizeof(w));
+        return w;
```

Comment:
I hate else after return.  Others it doesn't bother as much.  Move "return w" after the if and you can make us both happy :)